web/pf3/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "k-charted-pf3",
+  "version": "0.1.1",
+  "description": "Dashboards and Charts library for Kubernetes, with React / PatternFly 3",
+  "repository": "https://github.com/kiali/k-charted",
+  "author": "Red Hat",
+  "license": "Apache-2.0",
+  "private": false,
+  "main": "dist/pf3/src/index.js",
+  "types": "dist/pf3/src/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "peerDependencies": {
+    "@types/react": ">=16.0.0",
+    "react": ">=16.0.0"
+  },
+  "dependencies": {},
+  "devDependencies": {
+    "@babel/core": "~7.4.3",
```

Comment:
At some point are we going to lock down these dependency versions?

Reply:
I don't know. This is the approach we have in Kiali UI, but after having experimenting, I'm questioning this approach for two reasons:

- `yarn.lock` is also ensuring they are locked
- many transitive dependencies have tilde or caret versions, hence even our "locked" dependencies are not so much locked in the end.